dist/stat_data.py

```diff
@@ -518,11 +518,20 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Dhandle statistics
     ##########################################
+    # dh_conn_handle_count = The sum of the next four.
```

Comment:
Can those stats be sorted alphabetically please? I will file a ticket so `s_all` does it for us.

Reply:
WT-13142

Reply:
What kind of sorting do you mean? Sorted by name or description, e.g. by  dh_conn_handle_count or "connection data handles currently active"?

Note, the content of statistics files WiredTigerStat.*.* are already sorted by description regardless of the order here.

Reply:
By name would make more sense to me.